diff --git a/node_modules/@ray-js/babel-preset-standard/index.js b/node_modules/@ray-js/babel-preset-standard/index.js
index 0fcd119..538bf81 100644
--- a/node_modules/@ray-js/babel-preset-standard/index.js
+++ b/node_modules/@ray-js/babel-preset-standard/index.js
@@ -17,7 +17,7 @@ function preset(api, presetOption) {
           platform: presetOption.platform,
         },
       ],
-      jsx === 'react' && [require.resolve('@babel/preset-react')],
+      jsx === 'react' && [require.resolve('@babel/preset-react'),{ throwIfNamespace: false }],
     ].filter(Boolean),
     plugins: [
       jsx === 'preserve' && require.resolve('@babel/plugin-syntax-jsx'),
